src/utils/sshUtils.ts

```diff
@@ -36,19 +38,21 @@ export async function getSshKey(vmName: string, passphrase: string): Promise<str
 export async function configureSshConfig(vmti: VirtualMachineTreeItem, sshKeyPath?: string): Promise<void> {
     const sshConfigPath: string = join(sshFsPath, 'config');
     await fse.ensureFile(sshConfigPath);
-    let configFile: string = (await fse.readFile(sshConfigPath)).toString();
 
     // If we find duplicate Hosts, we can just make a new entry called Host (2)...(3)...etc
     const hostName: string = await vmti.getIpAddress();
     let host: string = vmti.name;
 
-    if (configFile.includes(`Host ${vmti.name}`)) {
+    const configFile: string = (await fse.readFile(sshConfigPath)).toString();
+    // tslint:disable: no-unsafe-any
+    const sshConfig: SSHConfig = SSHConfig.parse(configFile);
```

Comment:
It's sad that there's no type safety in that package 😞 Would it be better to manually define the type? Something like this:

```ts
// tslint:disable-next-line:no-any
type SSHConfigType = { find(opts?: {}): any; append(opts: any): SSHConfig };

// tslint:disable-next-line:no-unsafe-any
const sshConfig: SSHConfigType = SSHConfig.parse(configFile);
```

It may not be any better than wrapping all the package use in a `no-unsafe-any` exception like you originally had

Reply:
I started this, but it felt like a little bit overkill.  I guess we can wait to see what the boss says.

Reply:
I guess I'm a bit hesitant to use a package that doesn't even have typescript types. What does the Remote SSH extension do? I'm sure they have to parse ssh config files, too - did they manually implement it, are they using the same package, or did they find a better one?

Reply:
Nice idea.  Remote SSH extension defined their own typings for the remote-ssh npm package here: https://github.com/microsoft/vscode-remote-ssh/blob/186fbac18b4859ab70f2c7332b5f608a49e45dc7/open-ssh-remote/src/typings/ssh-config.d.ts